src/testing/tmp_tigerbeetle.zig

```diff
@@ -19,6 +19,12 @@ port_str: stdx.BoundedArray(u8, 8),
 
 tmp_dir: std.testing.TmpDir,
 
+// A separate thread for reading process stderr without blocking it. The process must be terminated
+// before stopping the StreamReader.
+//
+// StreamReader echoes process' stderr unless explicitly instructed otherwise. Because
```

Comment:
"Because" is missing the explanation :wink:

Reply:
Too bad I forgot it, as I no longer remeber what I wanted to write there, just removed the `Because`